apps/ensindexer/src/plugins/basenames/handlers/Registrar.ts

```diff
@@ -34,79 +31,32 @@ export default function ({
     registrarManagedName: "base.eth",
   });
 
-  const registrarManagedNode = namehash("base.eth");
-
   // support NameRegisteredWithRecord for BaseRegistrar as it used by Base's RegistrarControllers
   ponder.on(namespace("BaseRegistrar:NameRegisteredWithRecord"), async ({ context, event }) => {
     await handleNameRegistered({
       context,
-      event: {
-        ...event,
-        args: {
-          ...event.args,
-          labelHash: tokenIdToLabelHash(event.args.id),
-        },
-      },
+      event: { ...event, args: { ...event.args, labelHash: tokenIdToLabelHash(event.args.id) } },
     });
   });
 
   ponder.on(namespace("BaseRegistrar:NameRegistered"), async ({ context, event }) => {
     await handleNameRegistered({
       context,
-      event: {
-        ...event,
-        args: {
-          ...event.args,
-          labelHash: tokenIdToLabelHash(event.args.id),
-        },
-      },
+      event: { ...event, args: { ...event.args, labelHash: tokenIdToLabelHash(event.args.id) } },
     });
   });
 
   ponder.on(namespace("BaseRegistrar:NameRenewed"), async ({ context, event }) => {
     await handleNameRenewed({
       context,
-      event: {
-        ...event,
-        args: {
-          ...event.args,
-          labelHash: tokenIdToLabelHash(event.args.id),
-        },
-      },
+      event: { ...event, args: { ...event.args, labelHash: tokenIdToLabelHash(event.args.id) } },
     });
   });
 
   ponder.on(namespace("BaseRegistrar:Transfer"), async ({ context, event }) => {
-    // base.eth's BaseRegistrar uses `id` instead of `tokenId`
-    const { id: tokenId, from, to } = event.args;
-
-    const labelHash = tokenIdToLabelHash(tokenId);
-
-    if (event.args.from === zeroAddress) {
-      // Each domain must reference an account of its owner,
-      // so we ensure the account exists before inserting the domain
-      await upsertAccount(context, to);
-      // The ens-subgraph `handleNameTransferred` handler implementation
-      // assumes an indexed record for the domain already exists. However,
-      // when an NFT token is minted (transferred from `0x0` address),
-      // there's no domain entity in the database yet. That very first transfer
-      // event has to ensure the domain entity for the requested token ID
-      // has been inserted into the database. This is a workaround to meet
-      // expectations of the `handleNameTransferred` subgraph implementation.
-      await context.db
-        .insert(schema.domain)
-        .values({
-          id: makeSubdomainNode(labelHash, registrarManagedNode),
-          ownerId: to,
-          createdAt: event.block.timestamp,
-        })
-        // ensure existing domain entity in database has its owner updated
-        .onConflictDoUpdate({ ownerId: to });
-    }
-
```

Comment:
Awesome to see this bit gone :) It always felt like a forced patch, so happy to see it removed.